tests/plugins/test_koji_import.py

```diff
@@ -1303,16 +1304,21 @@ def test_koji_import_add_help(self, tmpdir, os_env, expect_result):
          "application/vnd.docker.distribution.manifest.v2+json",
          "application/vnd.docker.distribution.manifest.list.v2+json"],
     ])
-    def test_koji_import_set_media_types(self, tmpdir, os_env, version):
+    def test_koji_import_set_media_types(self, tmpdir, os_env, has_pulp_sync, version):
         session = MockedClientSession('')
         tasker, workflow = mock_environment(tmpdir,
                                             name='ns/name',
                                             version='1.0',
                                             release='1',
                                             session=session)
-        build_info = BuildInfo(media_types=version)
-        orchestrate_plugin = workflow.plugin_workspace[OrchestrateBuildPlugin.key]
-        orchestrate_plugin[WORKSPACE_KEY_BUILD_INFO]['x86_64'] = build_info
+        if has_pulp_sync:
+            if version:
+                workflow.exit_results[PLUGIN_PULP_PULL_KEY] = ("acbdef1234", json.dumps(version))
+        else:
+            build_info = BuildInfo(media_types=version)
+            orchestrate_plugin = workflow.plugin_workspace[OrchestrateBuildPlugin.key]
+            orchestrate_plugin[WORKSPACE_KEY_BUILD_INFO]['x86_64'] = build_info
```

Comment:
No need for this entire block to be an `else`, but the media_types could be set to some canary value like `['look', 'elsewhere']` if `has_pulp_sync`.